scripts/remove-hooks.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
```

Comment:
Perhaps you can add a comment at the top of the file that this script is used for moving from git-validate to husky and it's about removing the git-valide hooks if they exist.

Reply:
And perhaps adding this information also to the commit message, so when browsing the history it's clear why a special script is needed.